<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QScrollArea" name="scrollArea_Left">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>69</width>
          <height>244</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QTreeView" name="treeView">
       <property name="minimumSize">
        <size>
         <width>199</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>600</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox_widgets"/>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_infoTypes">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea_Right">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>144</width>
             <height>214</height>
            </rect>
           </property>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>502</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDevices">
    <property name="title">
     <string>Devices</string>
    </property>
    <addaction name="actionCom_Device"/>
    <addaction name="actionSocket"/>
    <addaction name="actionServer"/>
   </widget>
   <widget class="QMenu" name="menuNodes">
    <property name="title">
     <string>Nodes</string>
    </property>
    <addaction name="actionAdd_Node"/>
    <addaction name="actionClear_all"/>
    <addaction name="actionSave_Settings"/>
    <addaction name="actionLoad_Settings"/>
   </widget>
   <addaction name="menuDevices"/>
   <addaction name="menuNodes"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCom_Device">
   <property name="text">
    <string>Com Device</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D, Ctrl+C</string>
   </property>
  </action>
  <action name="actionSocket">
   <property name="text">
    <string>Socket </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D, Ctrl+S</string>
   </property>
  </action>
  <action name="actionAdd_Node">
   <property name="text">
    <string>Add Node</string>
   </property>
  </action>
  <action name="actionClear_all">
   <property name="text">
    <string>Clear all</string>
   </property>
  </action>
  <action name="actionSave_Settings">
   <property name="text">
    <string>Save Settings</string>
   </property>
  </action>
  <action name="actionLoad_Settings">
   <property name="text">
    <string>Load Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
